hushline/templates/directory.html

```diff
@@ -169,4 +113,4 @@ <h3>{{ username.display_name or username.username }}</h3>
 <script src="{{ url_for('static', filename='js/directory_verified.js') }}"></script>
 {% else %}
 <script src="{{ url_for('static', filename='js/directory.js') }}"></script>
-{% endif %} {% endblock %}
+{% endif %} {% endblock %}
```

Comment:
Nit: Can you configure your IDE to leave (and/or add) trailing newlines since that's part of the POSIX standard

Reply:
✅